bin/install.sh

```diff
@@ -401,9 +401,7 @@ docker load --input images/waffle.io-migrations.tar
 echo -ne '[##################         ] (66%)\r'
 docker load --input images/waffle.io-rally-integration.tar
 echo -ne '[#####################      ] (77%)\r'
-docker load --input images/waffle.io-admin.tar
-echo -ne '[########################   ] (88%)\r'
-docker load --input images/nginx-proxy.tar
+docker load --input images/takeout-nginx.tar
```

Comment:
Yeah, I think this should be called `waffle-nginx` so the naming convention is consistent.